osprey/utils.py

```diff
@@ -61,10 +61,14 @@ def replace_urls(config, outdir):
             local_file.write(r.content)
             filedata[i] = filedata[i].replace(url, local_file.name)
 
-    with open(config, "w") as write_config:
+    config_filename = config_file.split("/")[-1]
```

Comment:
What is the anticipated `config_file` name here? What happens if it is not in the format we expect?

Reply:
Original file names are maintained. For example, if `https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/fileServer/datasets/RVIC/sample_flow_parameters.nc` is the input, the function will return `path/to/temporary/directory/sample_flow_parameters.nc`

Reply:
> What happens if it is not in the format we expect?

Could you give an example for this?

Reply:
I'm just wondering how this code will handle a bad path or a bad url. At what point will it be caught?

Reply:
Currently, `osprey` does not handle those. It only passes the configuration files to RVIC modules and let them handle those.

Reply:
I forgot that the `rvic` package does basically everything for us already. In that case we don't have to worry about this.